src/App.js

```diff
@@ -1,24 +1,73 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react'
+import BarList from './components/BarList'
+import './App.css'
+import axios from 'axios'
+import UserInput from './components/UserInput'
 
 function App() {
+  const [barList, setBarList] = useState([])
+  const [currentLocation, setCurrentLocation] = useState({})
+  const [locationUnavailable, setLocationUnavailable] = useState(true)
+
+  const baseURL = new URL('https://api.openbrewerydb.org/v1/breweries')
+  
+  async function fetchData() {
+    if ("geolocation" in navigator) {
+      const location = await getLocation()
+      await findBarsHandler(location)
+    } else {
+      await findBarsHandler()
+    }
+  }
+
+  async function getLocation() {
+    const promise = new Promise((resolve, reject) => {
+      navigator.geolocation.getCurrentPosition(resolve, reject)
+    })
+    try {
+      const {coords: location} = await promise
+      setCurrentLocation(location)
+      setLocationUnavailable(false)
+      return location
+    } catch (error) {
+      console.log(error)
+    }
+  }
+  
+  async function findBarsHandler(location=currentLocation) {
+    try {
+      if (!!location.latitude) {
```

Comment:
Since this is a number you don't want to use boolean coercion or truthy/falsy checks as 0 can be a valid value here. 

It'd probably be better to explicitly check if a valid number exists in the property.
```suggestion
      if (!Number.isNaN(location?.latitude)) {
```

Reply:
For additional reading, look into `optional chaining` (`?.`).